### ISSUE TITLE
what is stop argument

### ISSUE THREAD

**Issue by kokangit** [knilink/tamperpilot#1]
Thanks for this tool, really helpful!
Can you please elaborate on the --stop argument. What is it used?

**Comment by knilink** [comment]
it's a token sequences to tell when to stop generating [completions-create-stop](https://platform.openai.com/docs/api-reference/completions/create#completions-create-stop)

can be useful with some models, such as `codegemma` that uses special stop token `<|file_separator|>` which may unexpectedly included in the completion result like `    age = current_year - birth_year     <|file_separator|>`.

basically you can just omit `--stop` args and use built-in stop until you came across with situation like above.
